### PR TITLE
JAMES-2768 Make DelaysAndMaxRetry logs more clear

### PR DESCRIPTION
**Pull request by trantienduchn**
https://issues.apache.org/jira/browse/JAMES-2768

It was printing `Duration.toString()` - hard to read, not the milliseconds.

### CODE REVIEW

File: server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/remote/delivery/DelaysAndMaxRetry.java [hunk 1]
```diff
@@ -64,12 +64,13 @@ private static DelaysAndMaxRetry getDelaysAndMaxRetry(int intendedMaxRetries, in
     private static DelaysAndMaxRetry addExtraAttemptToLastDelay(int intendedMaxRetries, int extra, List<Delay> delayTimesList) throws MessagingException {
         if (delayTimesList.size() != 0) {
             Delay lastDelay = delayTimesList.get(delayTimesList.size() - 1);
-            LOGGER.warn("Delay of {} msecs is now attempted: {} times", lastDelay.getDelayTime(), lastDelay.getAttempts());
+            Duration lastDelayTime = lastDelay.getDelayTime();
+            LOGGER.warn("Delay of {} msecs is now attempted: {} times", lastDelayTime.toMillis(), lastDelay.getAttempts());
```
Comment (mbaechler) [inline server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/remote/delivery/DelaysAndMaxRetry.java]: I wouldn't have express the delay in msecs. Duration may have a human-readable formater?

Comment (trantienduchn) [reply server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/remote/delivery/DelaysAndMaxRetry.java]: The Duration.toString() returns a string kind of “P80S...” something.... It overrides toString() method, but impossible to understand. I will use some approprirate duration formaters

**Comment by Arsnael** [comment]
Crashed on step 15 `Deploying Guice JPA with data provision` (all other steps were green)

Can consider tests being :green_apple:

File: server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/remote/delivery/DelaysAndMaxRetry.java (hunk 1 above)
Comment (chibenwa) [reply server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/remote/delivery/DelaysAndMaxRetry.java]: Can we use http://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/time/DurationFormatUtils.html ?

**Comment by trantienduchn** [comment]
Before:
```
Delay of PT8M20S msecs is now attempted: 1 times
```
After:
```
Delay of 00:08:20.000 is now attempted: 1 times
```

**Comment by trantienduchn** [comment]
After 2: Using DurationFormatUtils.formatDurationWords() to have a more clear log
```
Delay of 8 minutes 20 seconds is now attempted: 1 times
```

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**